### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by evildmp**

### CODE REVIEW

File: docs/how_to/apphooks.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ this case, you can create a normal django CMS page without any content of its
 own, and attach the news application to the page; the news application's content
 will be delivered at the page's URL.
 
-To create an apphook place a ``cms_apps.py`` in your application. And in it write
+To create an Apphook place a ``cms_app.pys`` in your application. And in it write
```
Comment (yakky) [inline docs/how_to/apphooks.rst]: @evildmp this renaming changes ``cms_app.py`` to ``cms_apps.py``, the documentation has been changed accordingly

**Comment by evildmp** [comment]
That's fine!

**Comment by yakky** [comment]
So will you edit that?

**Comment by evildmp** [comment]
Oh I see, yes of course, sorry. Not sure what happened there.

**Pull request merged**